packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -1091,7 +1092,7 @@ class PageTransitionsTheme with Diagnosticable {
 
   static const Map<TargetPlatform, PageTransitionsBuilder> _defaultBuilders =
       <TargetPlatform, PageTransitionsBuilder>{
-        TargetPlatform.android: ZoomPageTransitionsBuilder(),
+        TargetPlatform.android: PredictiveBackPageTransitionsBuilder(),
```

Comment:
will this be a breaking change? I am not too familiar with the process of changing page transition. so maybe @QuncCccccc can chime in.

An alternative is some how expose the predicitiveback enabled setting to framework side and conditionally return PredictiveBackPageTransitionsBuilder or ZoomPageTransitionsBuilder.

Reply:
That's actually what it does internally:

https://github.com/flutter/flutter/blob/8d2c9782e334e5f14066b586a6e6cac916b17608/packages/flutter/lib/src/material/predictive_back_page_transitions_builder.dart#L73-L93

So this PR moves us from Zoom to FadeForwards by default, and when doing a predictive back gesture, it will use the _PredictiveBackSharedElement transition by default.

Can you verify that we're good to make FadeForwards the default @QuncCccccc?

Reply:
And it will always use that fallback aslong as you don't opt in in the AndroidManifest.xml, right?

Reply:
Yes that's right.